### DISCUSSION TITLE
Is there a reason this package uses intervention/image 2 and not 3 ?

### DISCUSSION THREAD

**Discussion by CharlieEtienne** [joshembling/image-optimizer#25] [General]
Hi,

I was already using https://github.com/Intervention/image-laravel in my project, which requires intervention/image ^3

Your package requires intervention/image ^2.7 so I can't use it...

**Comment by joshembling** [comment]
@CharlieEtienne Yes https://github.com/joshembling/image-optimizer/issues/16

If this has changed recently, I will look to update

**Reply by CharlieEtienne** [reply]
It seems Filament now supports `spatie/laravel-medialibrary` v11 [since Dec 17, 2023](https://github.com/filamentphp/spatie-laravel-media-library-plugin/commit/386775c13dab597df57b9b02fcb3cc1535700c97)
